sound/core/pcm_native.c

```diff
@@ -2897,6 +2897,8 @@ static snd_pcm_sframes_t snd_pcm_rewind(struct snd_pcm_substream *substream,
 
 	if (frames == 0)
 		return 0;
+	if (substream->runtime->info & SNDRV_PCM_INFO_NO_REWINDS)
+		return 0;
```

Comment:
Heh, didn't note this comment in pcm_native.c -- "  /* NOTE: we return zero for errors because PulseAudio gets depressed upon receiving an error". :) So it seems this is sufficient to keep PA not depressed, nice.